# Translations template for invenio-records-rest.
# Copyright (C) 2025 CERN
# This file is distributed under the same license as the
# invenio-records-rest project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2025.
# 
# Translators:
# Matti Lassila <matti.lassila@gmail.com>, 2025
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: invenio-records-rest 3.1.0\n"
"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
"POT-Creation-Date: 2025-07-08 19:51+0000\n"
"PO-Revision-Date: 2025-04-03 11:38+0000\n"
"Last-Translator: Matti Lassila <matti.lassila@gmail.com>, 2025\n"
"Language-Team: Finnish (https://app.transifex.com/inveniosoftware/teams/23537/fi/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.17.0\n"
"Language: fi\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: invenio_records_rest/config.py:262
msgid "Best match"
msgstr "Paras osuma"

#: invenio_records_rest/config.py:268
msgid "Most recent"
msgstr "Uusimmat"

#: invenio_records_rest/errors.py:51
msgid "Invalid query syntax."
msgstr "Virheellinen kyselyn syntaksi."

#: invenio_records_rest/errors.py:67
#, python-format
msgid "Style %(style)s could not be found."
msgstr "Tyyliä %(style)s ei löytynyt."

#: invenio_records_rest/errors.py:93
msgid "PID does not exist."
msgstr "PID:tä ei ole olemassa."

#: invenio_records_rest/errors.py:105
msgid "PID is not registered."
msgstr "PID ei ole rekisteröity."

#: invenio_records_rest/errors.py:117
msgid "PID has been deleted."
msgstr "PID on poistettu."

#: invenio_records_rest/errors.py:129
#, python-format
msgid "No object assigned to %(pid)s."
msgstr "PID:lle %(pid)s ei ole liitetty objektia."

#: invenio_records_rest/errors.py:142
#, python-format
msgid "Invalid redirect - pid_type %(pid_type)s endpoint missing."
msgstr ""
"Virheellinen uudelleenohjaus – pid_type %(pid_type)s päätepiste puuttuu."

#: invenio_records_rest/errors.py:160
#, python-format
msgid "PID %(pid)s could not be resolved."
msgstr "PID:tä %(pid)s ei voitu ratkaista."

#: invenio_records_rest/errors.py:174
#, python-format
msgid "Unsupported media type \"%(content_type)s\"."
msgstr "Ei tuettu mediatyyppi \"%(content_type)s”."

#: invenio_records_rest/errors.py:188
msgid "Could not load data."
msgstr "Tietoja ei voitu ladata."

#: invenio_records_rest/errors.py:200
msgid "Could not patch JSON."
msgstr "JSONia ei voitu päivittää (patch)."

#: invenio_records_rest/errors.py:213
#, python-format
msgid "Missing %(ctx_field)s context."
msgstr "Konteksti %(ctx_field)s puuttuu."

#: invenio_records_rest/errors.py:228
#, python-format
msgid "No completions requested.%(options)s"
msgstr "Täydennyksiä ei pyydetty.%(options)s"

#: invenio_records_rest/errors.py:243
#, python-format
msgid "Validation error: %(error)s."
msgstr "Validointivirhe: %(error)s."

#: invenio_records_rest/errors.py:257
msgid "An internal server error occurred when handling the request."
msgstr "Pyynnön käsittelyssä tapahtui sisäinen palvelinvirhe."

#: invenio_records_rest/views.py:66
msgid "The syntax of the search query is invalid."
msgstr "Kyselyn syntaksi on virheellinen."

#: invenio_records_rest/views.py:441
msgid "The query parameters from and page must not be used at the same time."
msgstr "Kyselyn parametreja 'from' ja 'page' ei saa käyttää samanaikaisesti."

#: invenio_records_rest/views.py:478
msgid "Invalid pagination parameters."
msgstr "Virheelliset sivutusparametrit."

#: invenio_records_rest/views.py:515
#, python-format
msgid "Maximum number of %(count)s results have been reached."
msgstr "Tulosten enimmäismäärä %(count)s on saavutettu."
